docs/UserGuide.md

```diff
@@ -227,15 +226,16 @@ Scroll down to the bottom and click on `recruitIn.jar`.
 </div>
 
 ### Viewing help : `help`
+[Table of contents](#table-of-contents)
 
 Shows a message explaining how to access the help page.
```

Comment:
Perhaps we should standardise how we write the explanation portion of every command? I saw example UGs from the industry and they tend to start with "The `help` command", followed by "shows a message explaining how to access the help page", so my entire section I followed that style.